dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/JaxwsConduitConfigActivator.java

```diff
@@ -0,0 +1,82 @@
+/**
```

Comment:
There are several reasons I did this: 

1. I had to split the original CXF classes as written. The main reason for splitting the classes is that the the way in which the jaxws-2.x integration layer adds it's sub-classed or Open-Liberty specific extensions to CXF's Bus. That's using via Bus Extensions, and the way that our integration layer registers these extensions requires the extension to have a default constructor. I split the single CXF Activator class into the ConduitConfigurer and ConduitConfigurerService because of the need for a default constructor. 
2. CXF uses an OSGi ManagedService for the configuration application to it's Conduit object, and builds that MangedService with the bundle context via the bundle activation method. I created the JaxWsConduitConfigActivator for two functions: to serve as a declarative service and get the BundleContext needed for the ManagedService, but also to act as an ExtensionProvider so we can register the our ConduitConfigurer class as an HTTPConduitConfigurer extension on the Bus. Of the three classes this is probably the most functionally different from the original CXF class. I "lifted" the start() stop() methods, but it's role as an DS Component is unique to Open-Libertry
3. The CXF Activator class relies on another CXF BundleActivator class: the HTTPTransportsActivator class. The JAX-RS team disables that class from being turned on (see com.ibm.ws.cxf.transports.http.NoOpActivator), which means the original CXF class is unusable at runtime by our JAX-WS integration layer. This refactoring allows our integration layer to get around disabling the rest of CXF transport layer's bundle activation.
4.  The HttpConduitConfigApplier is a private class, so opening it up to would be a change to CXF which I was trying to avoid. If a change to making the class public is fine, then I do think this class could be dropped entirely.

Reply:
Then my concern is that we do not have the open liberty copyright.  Should it have both the open liberty copyright and the CXF apache one?